internal/io/io.go

```diff
@@ -94,7 +94,7 @@ func CloseFileIgnore(file *os.File) {
 }
 
 func ExcludeTestFilter() filter.LoaderFilter {
-	return func(abspath string, info files.FileInfo, depth int) bool {
+	return func(_ string, info files.FileInfo, _ int) bool {
```

Comment:
Sorry, unrelated change. I had just happened to look into how the bundle is loaded...